tests/contrib/operators/test_postgres_to_gcs_operator.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from __future__ import absolute_import
```

Comment:
Same question here. What's this needed for?

Reply:
I'm importing the full stack of `from __future__`s here because I find it weird to only import the one I need, `unicode_literals`. This matches the style of [other `from __future__ ...` imports being used in airflow currently](https://github.com/apache/incubator-airflow/search?utf8=%E2%9C%93&q=from+__future__&type=).

For the test, I need `unicode_literals` so that my literal string representations in the constants are consistently and explicitly encoded (byte strings) or decoded (unicode strings). Without it, the py2.7 interpreter would identify a literal `"foo"` as encoded while the py3.4 interpreter would identify it as decoded.

Being explicit about this is important for testing the two weird 2.7/3.4 inconsistencies that the PostgresToGoogleCloudStorage operator deals with.
1. [In py2.7 the psycopg2 cursor will return encoded bytes while the py3.4 one will return decoded strs.](http://initd.org/psycopg/docs/usage.html#strings-adaptation).
2. In py2.7, json dumping creates encoded bytes and in py3.4 it returns decoded bytes (this is also the root problem behind [AIRFLOW-1613](https://issues.apache.org/jira/browse/AIRFLOW-1613)).